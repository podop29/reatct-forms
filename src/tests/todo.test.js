import React from "react"
import {render} from "@testing-library/react"
import Todo from "../todo"

it("renders witout crashing", ()=>{
    render(<Todo/>)
})


//snapshot test
it("matches snapshot", ()=>{
    const {asFragment} = render(<Todo/>);
    expect(asFragment()).toMatchSnapshot()
})