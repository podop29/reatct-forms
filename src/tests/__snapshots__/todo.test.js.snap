// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="todo"
  >
    <h3 />
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
